src/main/java/frc/robot/networktables/DebugEntry.java

```diff
@@ -47,9 +47,11 @@ public DebugEntry(T defaultValue, String name, SubsystemBase subsystem) {
           networkEntry = networkTab.add(name, defaultValue).getEntry();
           entries.put(name, networkEntry);
         } else {
-          networkEntry = entries.get(name);
-          DriverStation.reportWarning(
-              "Duplicate ShuffleboardEntry on " + networkTab.getTitle() + " tab: " + name, false);
+          // networkEntry = entries.get(name);
+          String errorMessage =
+              "Duplicate ShuffleboardEntry on " + networkTab.getTitle() + " tab: " + name;
+          DriverStation.reportError(errorMessage, false);
```

Comment:
REmoving this message doesn't cause a test error.

I think that the throw is skipping part of your test.  

Make sure to change the pieces of the code to cause a failure and observe that the test fails when it should